frontend/src/views/ps.categoriaAssunto/CategoriaAssuntoCriarEditar.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <MigalhasDePão class="mb1" />
+  <div class="flex spacebetween center mb2">
+    <h1> <span v-if="!categoriaAssuntoId">Novo</span> Assunto</h1>
```

Comment:
Por que `<span>`, @GustavoFSoares ? Será formatado diferente? Se não for, não vejo motivo para criar outro elemento no DOM. Para isso, existe o `<template>`.